### PR TITLE
Rev1 track

### PR DESCRIPTION
**Pull request by matthewfl**
cleaning of some tests and sending more data along

### CODE REVIEW

File: src/balanced.js [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
+    language:navigator.userLanguage || navigator.language,
+    kp: 0,
+    cli: 0,
+    loaded: (new Date) * 1,
+    screen_width: screen.width,
+    screen_length: screen.height,
+    hist: window.history.length
 };
 
+var cookie = document.cookie.match(/__b=([a-zA-Z0-9\-!\.]+)/);
```
Comment (mjallday) [inline src/balanced.js]: this looks like a function but it's all inline.

**Comment by mjallday** [comment]
Looks good apart from the failing travis build. Travis is complaining about a missing Chrome extension

```
ERROR [launcher]: Cannot start Chrome
    Can not find the binary google-chrome
    Please set env variable CHROME_BIN
Warning: Task "karma:unit" failed. Use --force to continue.
```

**Comment by matthewfl** [comment]
so apparently travis does not support chrome, so I am just removing that again

File: Gruntfile.js [hunk 2]
```diff
@@ -9,10 +9,10 @@ module.exports = function (grunt) {
         uglify: {
             js: {
                 options: {
-                    banner: '////\n// <%= pkg.name %>\n// version: <%= pkg.version %>\n// built: <%= grunt.template.today("yyyy-mm-dd") %>\n////\n\n',
-                    footer: '\n\n/*\n' + grunt.file.read('license.txt') + '\n*/',
-                    mangle: false,
-                    beautify: true,
+                    banner: '////\n// <%= pkg.name %>\n// version: <%= pkg.version %>\n// built: <%= grunt.template.today("yyyy-mm-dd") %>\n// https://github.com/balanced/balanced-js\n////\n\n',
+                    mangle: true,
```
Comment (mjallday) [inline Gruntfile.js:13]: no license?

Comment (matthewfl) [inline Gruntfile.js:13]: do you really want that, it was being added to the end and it just looked pointless

I think it is better just to link to the github repo if someone cares for the license?  maybe mention the copyright

if we are going to add it, then it should go at the top like every other javascript

Comment (mjallday) [inline Gruntfile.js:13]: it was always supposed to be at the top. the last person to work on this repo changed it for i don't know why.

**Pull request merged**